packages/xo-web/src/xo-app/home/backed-up-vms.js

```diff
@@ -0,0 +1,248 @@
+import _ from 'intl'
+import decorate from 'apply-decorators'
+import Link from 'link'
+import PropTypes from 'prop-types'
+import React from 'react'
+import renderXoItem, { Vm } from 'render-xo-item'
+import SortedTable from 'sorted-table'
+import { addSubscriptions, connectStore, createCompare } from 'utils'
+import { compact, every, find, isEmpty, map, omit, some, uniq } from 'lodash'
+import { createGetObjectsOfType } from 'selectors'
+import { createPredicate } from 'value-matcher'
+import { injectState, provideState } from 'reaclette'
+import { Text, XoSelect } from 'editable'
+import {
+  copyVms,
+  deleteVms,
+  editVm,
+  migrateVm,
+  migrateVms,
+  pauseVms,
+  restartVms,
+  snapshotVms,
+  startVms,
+  stopVms,
+  subscribeBackupNgJobs,
+  suspendVms,
+} from 'xo'
+
+const createCompareContainers = poolId =>
+  createCompare([c => c.$pool === poolId, c => c.type === 'pool'])
+
+const COLUMNS = [
+  {
+    name: _('name'),
+    itemRenderer: _ => <Vm id={_.id} link newTab />,
+    sortCriteria: _ => _.name_label,
+  },
+  {
+    name: _('formDescription'),
+    itemRenderer: vm => (
+      <Text
+        value={vm.name_description}
+        onChange={value => editVm(vm, { name_description: value })}
+        placeholder={_('vmHomeDescriptionPlaceholder')}
+        useLongClick
+      />
+    ),
+    sortCriteria: _ => _.name_description,
+  },
+  {
+    name: _('containersTabName'),
+    itemRenderer: (vm, { pools, hosts }) => {
+      let container
+      return vm.power_state === 'Running' &&
+        (container = hosts[vm.$container]) ? (
+        <XoSelect
+          compareContainers={createCompareContainers(vm.$pool)}
+          labelProp='name_label'
+          onChange={host => migrateVm(vm, host)}
+          placeholder={_('homeMigrateTo')}
+          useLongClick
+          value={container}
+          xoType='host'
+        >
+          <Link to={`/${container.type}s/${container.id}`}>
+            {renderXoItem(container)}
+          </Link>
+        </XoSelect>
+      ) : (
+        (container = pools[vm.$container]) && (
+          <Link to={`/${container.type}s/${container.id}`}>
+            {renderXoItem(container)}
+          </Link>
+        )
+      )
+    },
+  },
+]
+
+const ACTIONS = [
+  {
+    handler: vms => {
+      if (every(vms, _ => _.power_state === 'Halted')) {
```

Comment:
What if the VM is suspended or paused?

Reply:
Yes, the reboot can stay in the grouped actions but I think we should be able to both start *and* stop a suspended/paused VM.